src/include/sof/debug/debug.h

```diff
@@ -28,6 +27,18 @@
 #define DEBUG_GDB	0
 #endif
 
+#if CONFIG_DEBUG_LOCKS
+#define DEBUG_LOCKS	1
+#else
+#define DEBUG_LOCKS	0
+#endif
+
+#if CONFIG_DEBUG_LOCKS_VERBOSE
+#define DEBUG_LOCKS_VERBOSE	1
+#else
+#define DEBUG_LOCKS_VERBOSE	0
+#endif
+
```

Comment:
Why do you still need DEBUG_LOCKS and DEBUG_LOCKS_VERBOSE if you have the CONFIG_ version?